keps/prod-readiness/sig-storage/3314.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 3314
+alpha:
+  approver: "@johnbelamaric"
```

Comment:
I've been looking at this before as shadow, and John is overbooked. So let's switch to me with PRR approval on this one.

Reply:
Done!